tests/test_models/test_inference.py

```diff
@@ -185,6 +183,16 @@ def test_forecast_in_sample_full(model, transforms, example_tsds):
         (LinearMultiSegmentModel(), [LagTransform(in_column="target", lags=[2, 3])]),
         (ElasticPerSegmentModel(), [LagTransform(in_column="target", lags=[2, 3])]),
         (ElasticMultiSegmentModel(), [LagTransform(in_column="target", lags=[2, 3])]),
+        pytest.param(
+            DeepARModel(encoder_length=1, decoder_length=1, trainer_params=dict(max_epochs=1), lr=0.01),
+            [],
+            marks=pytest.mark.xfail(reason="PytorchForecasting nets need context and horizon in forecast", strict=True),
```

Comment:
You need to update the current branch with the changes from tsdataset_2, there if have already fixed passing the horizon to the model [see](https://github.com/tinkoff-ai/etna/blob/3e5d766aacd8cb156ef80cddae2fd57f0b6f88ff/etna/pipeline/pipeline.py#L58)

Reply:
This is all about test cases and helper functions.

This will be fixed in inference track. Current models works correctly, [see](https://github.com/tinkoff-ai/etna/pull/971/files#diff-e84b3fce6ece143c6b54251120ec58c35fc36adf04fc11a4d686602b4e01466dR58)